test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/Internal/ViewComponentTagHelperChunkVisitorTest.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.AspNetCore.Razor.Chunks;
 using Microsoft.AspNetCore.Razor.CodeGenerators;
 using Xunit;
+using System;
```

Comment:
This was necessary to allow compilation when GENERATE_BASELINES is [pre-processor] defined.  I removed and changed inline as necessary.